### ISSUE TITLE
Update README

### ISSUE THREAD

**Issue by hackerb9** [bgri/m100LE#34]
Before the next release, it would be good to update the README.md file. I do not think it will require much.

* [ ] Instructions for which files to download (M100.BA and WL20XX.CO)
* [ ] New section on non -Tandy Kyotronic sisters (NEC, K-85, Olivetti)
  * [ ] Different .BA file (same source .DO)
  * [ ] Same Word List files 
  * [ ] YY/MM/DD format
* [ ] Update details on the DATE entry screen
  * [ ] Default is today's Wordle
  * [ ] Allows ordinal day ("Day of Year")
  * [ ] Mention that the date must be typed exactly
  * [ ] Remove statement that a Y2K patched ROM is needed

**Comment by bgri** [comment]
Looks good. Yeah, given all the updates you've made (awesome work!!) it's almost a new program :) I'll take a pass at this.

**Comment by bgri** [comment]
Ok, first draft at an update. How's this? Feel free to tweak as needed -- I may have missed something as there was a lot updated :)
[README.md.zip](https://github.com/bgri/m100LE/files/9740344/README.md.zip)

Also, I'm thinking we can probably take the current version and .zip it up for a 0.m release once we get this readme done. Then move forward...

Thoughts?

**Comment by hackerb9** [comment]
I've made a new branch to experiment with edits. Let me know what you think of this https://github.com/hackerb9/M100LE/tree/documentation

I feel like it's not done, but I know that part of what's bugging me is things like telling people that they have to type the DATE *exactly* or it won't work. Sometimes I'd rather fix a program than admit to having a misfeature like that.

**Comment by bgri** [comment]
Nice! I like the organizational changes and the additional detail about the optimization. I'm totally good to go with this version. If you agree I'll zip up the whole shebang and pop it into the releases section. Nice work!!

As for the date, I guess we could look at putting in some sort of date picker routine. Display a month calendar and navigate using arrow keys etc. Though it would look nice and 'modern', it would likely add a lot of code. 

Or do a date verification before looking for the word list file? Something like:
`You entered Saturday, October 1, 2022. [C]ontinue or [E]dit?`

**Comment by hackerb9** [comment]
Oh, the thing that's bugging me about the date is that if you type 3/4/22, it gets confused.

Let's hold off briefly on packaging it up. I'm trying to finish up some documentation on how to transfer the files and in so doing I'm coming across small bugs. For example, the [A]GAIN option gave the system date as the default instead of the game you just played. I've fixed that (and a couple others) and will send a pull request. 

Also, did we ever confirm that CMPRSS.DO works on a NEC? I realize I never wrote up a howto for using it, so I'm doing that now.

**Comment by bgri** [comment]
Ah, do we want to put in some date checking to enforce the format? When looking at calendar dates, only accept MM/DD/YY?

Totally good on holding off for a bit.  And no, don't think we've tested CMPRSS.DO on a NEC. I'll do that tonight.

**Comment by bgri** [comment]
> Also, did we ever confirm that CMPRSS.DO works on a NEC? I realize I never wrote up a howto for using it, so I'm doing that now.

Yep. Tonight I've just loaded fresh copies of CMPRSS and WL2022. CMPRSS created a good wordlist .CO file that I spent way too much time testing.

It seems my 8201a is having a hardware issue or two. I have to spend some time and take a look at it.  It's overdue. I think I'm having a minor issue with the serial port, and there's now a bad LCD strip on one of the display controler panels. So I need to poke at it. But first, I should get my 8300 up and running so I can use it while this one is on the workbench :)

Compounding things, I've got REX#  and the [2nd bank 32k ram expansion card](https://bitchin100.com/wiki/index.php?title=PC-8201_BANK2_32kB_SRAM_card) installed. So there could be a lot going on there. Yay?

**Comment by hackerb9** [comment]
> > Also, did we ever confirm that CMPRSS.DO works on a NEC? I realize I never wrote up a howto for using it, so I'm doing that now.
> 
> Yep. Tonight I've just loaded fresh copies of CMPRSS and WL2022. CMPRSS created a good wordlist .CO file that I spent way too much time testing.
 
If by "testing", you mean actually playing the game and having fun, then yes, that happens to me, too. It's actually quite enjoyable in this form factor. 

When I want to play multiple games, to avoid seeing words that haven't happened yet, I usually go backwards in time, always selecting the previous (yesterday's) puzzle. It's a bit inconvenient, though. I've been thinking it would be nice to have a "[P]REVIOUS" option (maybe hit the `-` MINUS key at the end of the game). Of course, good idea or not, it'd wait until after 0.m no matter what. 

One thing that would be nice to get nailed down before the release is whether CMPRSS works over the serial port for the NEC computers. 


> It seems my 8201a is having a hardware issue or two. I have to spend some time and take a ld t it. It's overdue. I think I'm having a minor issue with the serial port, and there's now a bad LCD strip on one of the display controler panels. So I need to poke at it. But first, I should get my 8300 up and running so I can use it while this one is on the workbench :)
> 
> Compounding things, I've got REX# and the [2nd bank 32k ram expansion card ](https://bitchin100.com/wiki/index.php?title=PC-8201_BANK2_32kB_SRAM_card) installed. So there could be a lot going on there. Yay?

Oof. It's a good thing you've got more than one to tide you over until you can get that fixed. Did you say you also have a PC-8200 (as well as PC-82001A).

**Comment by bgri** [comment]
> If by "testing", you mean actually playing the game and having fun, then yes, that happens to me, too. It's actually quite enjoyable in this form factor.

Yes, heh, that's exactly what I mean by "testing". :)

> I've been thinking it would be nice to have a "[P]REVIOUS" option (maybe hit the `-` MINUS key at the end of the game).

I like the MINUS key thing. Automagically restart the game with the previous date. Nice quick shortcut.




> One thing that would be nice to get nailed down before the release is whether CMPRSS works over the serial port for the NEC computers.

For sure. I can try it once I get one of my 8300 off the bench.

> Did you say you also have a PC-8200 (as well as PC-82001A).

Yep. 8201 (Red). LCD Polarizer filter is detaching. Not full 32k RAM. Should be my backup unit if REX# will talk to it (have to test).
8300. On the bench tonight to swap out the memory battery and give it a good cleaning.
8201a. Not feeling so hot. Needs a bit of a thorough review. I think there may be some corrosion I missed a year or so ago.

I've been stocking up on projects for winter :)

**Comment by bgri** [comment]
<img width="774" alt="image" src="https://user-images.githubusercontent.com/14062627/196064104-47e5af2e-216a-47f2-bbb4-b679ab11770c.png">

More 'testing' on the 8201.

<img width="774" alt="image" src="https://user-images.githubusercontent.com/14062627/196064135-d4cf822c-b838-4b71-af30-510f37c2ad7b.png">

<img width="774" alt="image" src="https://user-images.githubusercontent.com/14062627/196064151-4ff2307d-547b-49cd-a4f3-33c1729457cd.png">

... and once I get to connecting this to my WinBox, I should be able to test if NEC CMPRSS will do the needful :)

**Comment by hackerb9** [comment]
Thanks for the screenshots. That's great that the BSAVE kludge actually works on a genuine NEC. And it's satisfying to see that, even with the European style DATE$, the correct word is being picked. 

One thing: I don't understand why the system labels are still showing, though. Does `SCREEN 0,0` not turn them off on a NEC 8201?

**Comment by bgri** [comment]
Oh! I think I tested on an older version. I moved my REX# from my ailing 8201a to the 8201 and refreshed from a slightly older backup, so yeah, it's probably not the current one. I'll load it and test that today.

**Comment by bgri** [comment]
Yup, an older version. `SCREEN 0,0` works fine in this version.

**Comment by bgri** [comment]
Note to self: update images in readme to include new position of the Alphabet panel and images from different platforms.
